packages/webdriverio/webdriverio.d.ts

```diff
@@ -1,11 +1,10 @@
 /// <reference types="webdriverio/webdriverio-core"/>
 
-type BrowserObject = WebDriver.ClientOptions & WebDriver.ClientAsync & WebdriverIOAsync.Browser;
 type $ = (selector: string | Function) => Promise<WebdriverIOAsync.Element>;
 type $$ = (selector: string | Function) => Promise<WebdriverIOAsync.Element[]>;
 
 // Element commands that should be wrapper with Promise
-type ElementPromise = WdioOmit<WebdriverIO.Element, 'addCommand' | '$' | '$$'>;
+type ElementPromise = Omit<WebdriverIO.Element, 'addCommand' | '$' | '$$'>;
```

Comment:
We renamed this because it caused some conflicts with a typescript definition, has that been resolved?

Reply:
If we change minimum typescript version, then no. I've also removed our implementation of Omit.